infrastructure/resource_group_level/modules/storage.bicep

```diff
@@ -0,0 +1,421 @@
+param environment string
```

Comment:
Note that when deployed to a new environment, the files storage has 7 day soft delete enabled, unlike the existing environments. I could look to disable it or just leave it as a reasonable setting.

Reply:
happy to leave it as it is for now

Reply:
I've made is parameterisable so easy to change when we want to.